ecosystem/sep-0010.md

```diff
@@ -19,56 +19,56 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
 It involves three components:
-- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`), referred to as the home domain.
-- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain where the endpoint is hosted is referred to as the web auth domain. 
-- A client who will authenticate using a Stellar account.
+- The **TOML Server**: a server hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`). The ``WEB_AUTH_ENDPOINT`` must be hosted by the **Web Auth Server** as described below.
+- The **Web Auth Server**: a domain hosting an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the **TOML Server**'s domain, a sub-domain of the **TOML Server**'s domain, or any other domain.
```

Comment:
💡  I think TOML Server and Web Auth Server are being a little too prescriptive and distancing the definition of the participants from what matters for SEP-10. SEP-10 doesn't need you to have these servers in the modern sense of the word, but needs you to provides:
- A home domain that has a toml file at a URL at that domain.
- An web auth endpoint.

Whatever approach an implementer takes to providing those is out-of-scope, and making the home domain a participant is critical to the rest of the specification where the home domain is a thing.

```suggestion
- A **Home Domain**: a domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`).
- A **Web Auth Endpoint**: an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the **Home Domain**, a sub-domain of the **Home Domain**, or any other domain. 
- A **Web Auth Domain**: the domain hosting the **Web Auth Endpoint**.
```

Thoughts?

Reply:
I like the change from `TOML Server` to `Home Domain` because you're right, we use the word 'home domain' throughout the doc and in request parameters. 

However I think the change from `Web Auth Server` to `Web Auth Domain` & `Web Auth Endpoint` is a bit unnecessary. Unlike the home domain hosting a static file at a path, an actual API server is required to process authentication requests. I don't see the value of trying to de-couple the endpoint and domain of the web auth server.

Reply:
I see your point. How about we replace Web Auth Endpoint with Web Auth Server, but keep Web Auth Domain in the list because it is an important attribute elsewhere in the SEP?

Reply:
Hmm if the goal is to define a component for the `web_auth_domain` key mentioned in the SEP, then I think we should go with what you had: `Web Auth Endpoint` & `Web Auth Domain`, since the TOML uses `WEB_AUTH_ENDPOINT`.

However, I'm not sure it worth it to explicitly define a `Web Auth Domain`, since there are only two places where it would be used. Do you think the current phrasing is confusing?

(in the abstract)
> 1. The **Client Application** verifies that if the transaction has a Manage Data operation with key `web_auth_domain` that it has:
>       1. Source account set to the ``SIGNING_KEY`` from the **Home Domain**'s [SEP-1 stellar.toml](sep-0001.md).
>       1. Value set to the **Web Auth Server**'s domain.

(in the response)
> * `manage_data(source: server_account, key: 'web_auth_domain', value: web_auth_domain)`
>        * The source account is the **Home Domain**'s ``SIGNING_KEY``, the value is the domain of the **Web Auth Server**. It can be at most 64 characters.

Reply:
I see your point. It's not anymore ambiguous to refer to the domain of the Web Auth Server.

I also agree for Web Auth Endpoint so as to remain consistent with the stellar.toml, so there is one less new concept and we continue to reuse the existing concept.

Reply:
Ok, so the take-away is to change `Web Auth Server` to `Web Auth Endpoint`, but not use the additional `Web Auth Domain` term, since we can easily say `the domain of the **Web Auth Endpoint**`?

Reply:
Yup, that's how I interpreted what you were saying and I think that strikes the balance.